install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -90,3 +90,5 @@ INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_doma
 VALUES
 	('product_page_v2', 0, 'Experimental product page', 'Admin.Advparameters.Feature', 'This page is a work in progress. It includes new combination management features and other features under development (virtual products, packs, etc.)', 'Admin.Advparameters.Help');
 
+/* Disable AdminTabs tab */
+UPDATE `PREFIX_access` SET `view` = '0', `add` = '0', `edit` = '0', `delete` = '0' WHERE `id_profile` = '1' AND `id_tab` = (SELECT `id_tab` FROM `PREFIX_tab` t WHERE t.class_name = 'AdminTabs' LIMIT 1);
```

Comment:
Now that I re-read my PR. I'm wondering if this is necessary as we return false in the access method of the controller

Reply:
Both are unneeded, #19660 has already been reverted (maybe due to a bad merge).
```suggestion
```

Reply:
One last thing and it's ok for me :)